native/monogame/sdl/MGP_sdl.cpp

```diff
@@ -678,7 +682,7 @@ MGP_Window* MGP_Window_Create(
 
 	Uint32 flags = SDL_WINDOW_HIDDEN;// | SDL_WINDOW_FULLSCREEN_DESKTOP;
 
-#if MG_VULKAN
+#if defined(MG_VULKAN) || defined(MG_DIRECTX12)
 	flags |= SDL_WINDOW_VULKAN;
```

Comment:
should we use a VULKAN window when using directX? Seems a bit odd.

Reply:
I think this flag is required regardless of what it looks like.  I will double check it.